utils/rpms/daos.spec

```diff
@@ -247,6 +247,7 @@ getent group daos_admins >/dev/null || groupadd -r daos_admins
 %{_prefix}%{_sysconfdir}/vos_dfs_sample.yaml
 %{_prefix}%{_sysconfdir}/vos_size_input.yaml
 %{_libdir}/libdaos_common.so
```

Comment:
if common is the server one now, it should move to %files server

Reply:
And make the parallel change in `debian/daos{,-server}.install`.

Reply:
Does this still need to be addressed?

Reply:
Looks like merge screwed it up in the rpm-spec :( will add it back.

Reply:
Also I have changed daos.install and changelog for debian, since I did not add it as a server side library, do I have to  add anything to the server.install?